cmd/influxdb-writer/main.go

```diff
@@ -108,7 +108,9 @@ func main() {
 }
 
 func loadConfigs() (config, influxdata.HTTPConfig) {
-	chanCfgPath := mainflux.Env(envChanCfgPath, defChanCfgPath)
+	channelsCfgPath := mainflux.Env(envChannelsCfgPath, defChannelsCfgPath)
+	subtopicsCfgPath := mainflux.Env(envSubtopicsCfgPath, defSubtopicsCfgPath)
+	filters, _ := writers.LoadFiltersConfig(channelsCfgPath, subtopicsCfgPath)
```

Comment:
Don't ignore errors please

Reply:
Yes, you have fully right. This is why I have this question: https://github.com/mainflux/mainflux/pull/1072#discussion_r391892220

Reply:
@manuio I understand your response but I don't know what I need to do with this error because the loader isn't already loaded.

Reply:
@jonathandreyer I didn't see this, sorry. What I propose you is that you call `writers.LoadFiltersConfig` outside `loadConfigs` and after to instantiate the logger. You would be able to check the error and eventually log it with mainflux logger. If there is no error you can do `cfg.filterConfig = filters`